README.md

```diff
@@ -155,14 +165,126 @@ Follow these steps to run the project:
 
 ## Troubleshooting
 
-- If you encounter permission issues, ensure your `src/` directory and files have appropriate permissions
-- For Finch build issues, verify that the VM is running
-- Check that all required files are in the correct locations as per the project structure
+- If you encounter permission issues, ensure your `src/` directory and files have appropriate permissions.
+- For Finch build issues, verify that the VM is running.
+- Check that all required files are in the correct locations as per the project structure.
+
+## Conclusion
```

Comment:
This must remain in the blogpost. Remove it.

Reply:
Done